lib/jsonapi/error_view.ex

```diff
@@ -34,37 +37,43 @@ defmodule JSONAPI.ErrorView do
     |> serialize_error
   end
 
+  @spec missing_data_attributes_param :: map()
   def missing_data_attributes_param do
     "Missing attributes in data parameter"
     |> build_error(400, @crud_message, "/data/attributes")
     |> serialize_error
   end
 
+  @spec missing_data_id_param :: map()
   def missing_data_id_param do
     "Missing id in data parameter"
     |> build_error(400, @crud_message, "/data/id")
     |> serialize_error
   end
 
+  @spec missing_data_type_param :: map()
   def missing_data_type_param do
     "Missing type in data parameter"
     |> build_error(400, @crud_message, "/data/type")
     |> serialize_error
   end
 
+  @spec missing_data_param :: map()
   def missing_data_param do
     "Missing data parameter"
     |> build_error(400, @crud_message, "/data")
     |> serialize_error
   end
 
+  @spec missing_id :: map()
   def missing_id do
     "Missing id in data parameter"
     |> build_error(400, @crud_message, "/data/id")
     |> serialize_error
   end
 
-  def to_many_relationships_payload_for_standard_endpoint do
+  @spec too_many_relationships_payload_for_standard_endpoint :: map()
```

Comment:
I also fixed a typo here

Reply:
I wonder if we should continue to define "to_many" with a deprecation notice in the event folks are using these directly?

Reply:
My mistake. https://github.com/jeregrine/jsonapi/pull/198/commits/55dd0f2bf13a61b65302fabf1570318d5c492e3e reverts.

Reply:
My confusion sprang from the fact that both labels are appropriate. It is an error that pertains to a "to-many" relationship, and is triggered by their count being too high (i.e., there are "too many" of them).